src/main/scala/arcana/death/Repl.scala

```diff
@@ -0,0 +1,43 @@
+package arcana.world
+
+import scala.io.StdIn
+
+case class ReplOptions(typeInfo: Boolean)
+
+class Repl {
+  import Repl._
+
+  def start(options: ReplOptions) = {
+    printf(
+      "\u001b[32;1m" +
+        "Welcome to Arcana %s \n" +
+        "Type in expressions for evaluation.\n" +
+        "Type in exit or ctrl-d to quit\n" +
+        "\u001b[0m",
+      version
+    )
```

Comment:
You do not need to create another function to start repl. You can simply use constructor.
```suggestion
class Repl(options: ReplOptions) {
  import Repl._

  printf(
      "\u001b[32;1m" +
        "Welcome to Arcana %s \n" +
        "Type in expressions for evaluation.\n" +
        "Type in exit or ctrl-d to quit\n" +
        "\u001b[0m",
      version
    )
```